packages/build-utils/src/lambda.ts

```diff
@@ -47,6 +48,10 @@ interface GetLambdaOptionsFromFunctionOptions {
 
 export class Lambda {
   type: 'Lambda';
+  /**
+   * framework-specific lambda type label
```

Comment:
Can we document the possible options here? The current comment doesnt make any sense. Imagine you're seeing this for the first time.

Reply:
There aren't specific options, but I can add some examples: [`76a9761` (#8887)](https://github.com/vercel/vercel/pull/8887/commits/76a9761708ac0ea924ac04b6fab119acbea5cf36)